### PR TITLE
Write only TRU digits to AOD trig table...

### PR DESCRIPTION
**Pull request by peressounko**
- Write only TRU digits to AOD trig table; 
- Wrong DDL fix in rawConverter; 
- Remove Decompose fail message in clusterizer; 
- fix validity range

### CODE REVIEW

**Comment by peressounko** [comment]
Dear @shahor02, core team, could you please have a look at this PR? Thanks, Dmitri

File: Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h
Comment (shahor02) [inline Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h]: should not these methods be provided by the Cell classes? 
Do we need to wait for the reply of @mfasel ?

Comment (peressounko) [reply Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h]: Ideally both PHOS and EMCAL should provide consistent naming to avoid this Helper class. But this is current status. Comment from @mfasDa on filling this AOD table would be very welcome! Note that commit does not change anything from EMCAL side.

Comment (mfasDa) [reply Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h]: For EMCAL we currently plan only propagate STU L1 TimeSums to the AOD, not TRU digits, due to noise reason. TRU digits will only be used for QC where we will probably use the cell class as well for, the resolution should be sufficient.

In the cell class we defined (DataFormats/EMCAL/include/DataFormatsEMCAL/Cell.h, L155)
``` c++
  Bool_t getTRU() const { return isChannelType(ChannelType_t::TRU); }
 ``` 

Ideally we would combine the function names even though we have different implementations, for the 2 detectors, then a lot of the code can be combine with templates.


Not yet decided whether we will reuse the same class also for the L1 timesum from the STU.

Comment (shahor02) [reply Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h]: @mfasDa thanks for reply, just to be sure: it is ok to merge this PR with `isTRU(const o2::emcal::Cell& cell)` returning always `true`?

Comment (mfasDa) [reply Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h]: @shahor02 From how I understand the code no, as it would flag also FEC cells as TRU cells, it must return getTRU() from o2::emcal::Cell in the same way as it does for PHOS.

Comment (shahor02) [reply Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h]: @peressounko can you adapt the `isTRU(const o2::emcal::Cell& cell)` ?

Comment (peressounko) [reply Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h]: Dear @mfasDa, @shahor02, I added TRU selection for EMCAL too, please have a look. Please note, that in current O2 neither PHOS nor EMCAL do not have such selection, so I left it for EMCAL as is.

**Comment by peressounko** [comment]
@shahor02, checks passed. Probably we can merge

**Review by shahor02 — APPROVED** [review]

**Pull request merged**
